app/Transformers/CompanyTransformer.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace App\Transformers;
+
+class CompanyTransformer extends Transformer{
+
+    public function transform($companies, $options = null){
+        $data = [
+            'success' => true,
+            'data' => $companies,
```

Comment:
and this is not the way to work in transformer why have you written success status here ? i have asked you to write the table keys here